addon/mixins/click-outside.js

```diff
@@ -17,12 +17,12 @@ export default Mixin.create({
   setupOutsideClickListener: on('didInsertElement', function() {
     let clickHandler = this.get('handleOutsideClick').bind(this);
 
-    ['click', 'touchend'].forEach((e) => document.addEventListener(e, clickHandler));
+    ['click', 'touchstart'].forEach((e) => document.addEventListener(e, clickHandler));
```

Comment:
did you check if there are any other components using this mixin? This might affect other stuff